services/ui-src/src/verbiage/pages/wp/wp-alerts.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  initiatives: {
+    title: "Your form is not ready for submission",
```

Comment:
I think this should say "Your initiatives do not meet the topic requirements" based on the Figma. Otherwise looks great!

Reply:
oh, thanks for catching that!